AutocompleteClientTests/FW/Logic/Worker/ConstructorIOIntegrationTests.swift

```diff
@@ -11,7 +11,7 @@ import ConstructorAutocomplete
 
 // swiftlint:disable type_body_length
 class ConstructorIOIntegrationTests: XCTestCase {
-
+    
```

Comment:
Not sure where all these white spaces came from, but let's clean them up 👍

Reply:
Hmm not sure how these got added. Can't seem to get swift lint to work. Let's go over this during coding time next week.

Reply:
Tried to fix it manually, but may have made it worse. Does swift lint fix this for you?

Reply:
@stanlp1 Swift lint doesn't fix it for me, but it does show a little notification on the line if there is a lint error. If they don't show up, clicking the play button on the top left of Xcode to rebuild the project should do the trick.

Lmk if that doesn't work and we can take a look at it together.